linalgo/annotate/models.py

```diff
@@ -407,6 +409,33 @@ def __init__(
         self.annotator = Annotator(annotator)
         self.task = Task(task)
         self.reviewee = Annotator(reviewee)
-    
+
     def __repr__(self) -> str:
-        return f'Schedule::{self.type}::{self.status}'
\ No newline at end of file
+        return f'Schedule::{self.type}::{self.status}'
+
+
+@dataclass
+class Body:
+    text: str
+    extras: dict = field(default_factory=dict)
+
+    def __getattr__(self, name):
+        return self.extras.get(name, None)
+
+    def __setattr__(self, name, value):
+        if name in {"text", "extras"}:
+            super().__setattr__(name, value)
+        else:
+            self.extras[name] = value
+
+
+class BodyFactory:
```

Comment:
Why do we need a factory for the body? Also, I don't think it returns a `Body` instance, does it?